tarodb/asset_minting.go

```diff
@@ -487,6 +525,38 @@ func fetchAssetSprouts(ctx context.Context, q PendingAssetStore,
 	return taroCommitment, nil
 }
 
+// fetchAssetMetas attempts to fetch the asset meta reveal for each of the
+// passed assets.
+func fetchAssetMetas(ctx context.Context, db PendingAssetStore,
+	assets []*asset.Asset) (
+	map[asset.SerializedKey]*proof.MetaReveal, error,
+) {
```

Comment:
nit: weird wrapping.

Reply:
Yeah the main edge case in our style guide....